packages/www/components/StreamDetails/MultistreamTargetsTable/SaveTargetDialog.tsx

```diff
@@ -76,6 +76,9 @@ const composeIngestUrl = (ingestUrlStr: string, streamKeyStr: string) => {
   switch (ingestUrl.protocol) {
     case "rtmp:":
     case "rtmps:":
+      if (streamKeyStr.includes("?") && !streamKeyStr.endsWith("?")) {
```

Comment:
Can you add a comment explaining why this is necessary?

Reply:
Also, why shouldn't we add the `?` if the `streamKey` ends in a `?`? Don't we still want to keep it there so we'd add another question mark?

In the end, maybe the logic should be to always add a question mark? It will be removed anyway cause it's always the last, be it the only one or not. Unless I misunderstood how mist fumbles with it